geomesa-z3/src/main/scala/org/locationtech/geomesa/curve/LegacyYearZ3SFC.scala

```diff
@@ -0,0 +1,40 @@
+/***********************************************************************
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.curve
+
+import java.time.temporal.ChronoUnit
+
+import org.locationtech.geomesa.curve.NormalizedDimension.NormalizedTime
+
+/**
+ * Z3SFC with a legacy, incorrect max time value of 52 weeks. The max value is kept the same to ensure that
+ * index keys and query ranges are consistent. Any dates that exceed the original max time will be dropped into
+ * the last time bin, potentially degrading results for the last day or two of the year.
+ *
+ * @param precision bits used per dimension - note all precisions must sum to less than 64
+ */
+@deprecated("Z3SFC", "3.2.0")
+class LegacyYearZ3SFC(precision: Int = 21) extends {
+  // need to use early instantiation here to prevent errors in creating parent class
```

Comment:
Wow.  Tell me you hit this in a unit test and had to fix it:).

Reply:
indeed :)